lib/jerboa/format.ex

```diff
@@ -4,15 +4,27 @@ defmodule Jerboa.Format do
   """
 
   alias Jerboa.Format.{Meta, Header, Body, MessageIntegrity}
-  alias Jerboa.Format.{HeaderLengthError, BodyLengthError}
+  alias Jerboa.Format.{HeaderLengthError, BodyLengthError,
+                       First2BitsError, ChannelDataLengthError}
   alias Jerboa.Params
+  alias Jerboa.ChannelData
+
+  @typedoc """
+  Represents valid number of TURN channel
+  """
+  @type channel_number :: 0x4000..0x7FFF
```

Comment:
Unfortunately Elixir does not allow such constructs. We can't use module attributes in typespecs.